webservices/common/models/itemized.py

```diff
@@ -109,20 +112,28 @@ class ScheduleB(BaseItemized):
     recipient_zip = db.Column(db.String)
     disbursement_type = db.Column('disb_tp', db.String)
     disbursement_description = db.Column('disb_desc', db.String)
-    disbursement_date = db.Column('disb_dt', db.Date)
+    disbursement_date = db.Column('disb_dt', db.TIMESTAMP(timezone=False))
```

Comment:
The FEC is storing it as a Timestamp though, I could double check.  I originally tried assigning it to a Date in the api code, and was getting an error.  I can verify though.